ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/StandaloneExecutor.java

```diff
@@ -50,6 +67,9 @@
   private final String queriesFile;
   private final UdfLoader udfLoader;
   private final CountDownLatch shutdownLatch = new CountDownLatch(1);
+  private final Map<Class<? extends Statement>, Function<Pair<String, Statement>, Void>>
```

Comment:
Each `Function<Pair<blah>>` returns null - so why the use of `Function` and why pass the `Pair`? Use `Consumer<String, Statement>`.

I think this member can also be static and initialised once for the entire class using `ImmutableMap`.

Reply:
Changed it to ImmutableMap with `Consumer<Pair<String, Statement>>` for value. Consumer accepts one arg only so had to use `Pair`.
Set/Unset update `configProperties` so cannot be static.

Reply:
Cool but use `BiConsumer`...